aroquev/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public final class Comment {
+
+  private final long id;
+  private final String body;
+  private final String username;
+  private final long timestamp;
+  private final int upvotes;
```

Comment:
I think if you wanted to enforce a user can only upvote/downvote once per comment, you'd want to store a "Vote" as a separate entity. That's probably too much work for the starter project though.

Reply:
Okay, if I will try to implement it if I have the chance. As for now, do you think there is a good approach I could take given the fact that once anything is clicked on everything gets loaded once again?